File: Modul-6/Soal 5/PRAK605-2410817120019-NadyaRamadhani.py
ordo = int(input())
print("Matriks A")
MatA = []
for i in range(ordo):
    matriks = list(map(int, input().split()))
    MatA.append(matriks)
print("Matriks B")
MatB = []
for i in range(ordo):
    matriks = list(map(int, input().split()))
    MatB.append(matriks)
print("Matriks AXB")
MatAXB = []
for i in range(ordo):
    MatAXB.append([])
    for j in range(ordo):
        hasil = 0
        for matriks in range(ordo):
            hasil += MatA[i][matriks] * MatB[matriks][j]
        MatAXB[i].append(hasil)
for baris in MatAXB:
    for matriks in baris:
        print(matriks, end=" ")
    print()

File: Modul-6/Soal 5/PRAK605-2410817120019-NadyaRamadhani.c
#include <stdio.h>
int main(){
    int MatA[10][10], MatB[10][10], MatAXB[10][10];
    int ordo;
    scanf("%d", &ordo);
    printf("Matriks A\n");
    for(int i=0; i<ordo; i++){
        for(int j=0; j<ordo; j++){
            scanf("%d", &MatA[i][j]);
        }
    }
    printf("Matriks B\n");
    for(int i=0; i<ordo; i++){
        for(int j=0; j<ordo; j++){
            scanf("%d", &MatB[i][j]);
        }
    }
    for(int i=0; i<ordo; i++){
        for(int j=0; j<ordo; j++){
            int hasil = 0;
            for(int x=0; x<ordo; x++){
                hasil += MatA[i][x] * MatB[x][j];
            }
            MatAXB[i][j] = hasil;
        }
    }
    printf("Matriks AXB\n");
    for(int i=0; i<ordo; i++){
        for(int j=0; j<ordo; j++){
            printf("%d ", MatAXB[i][j]);
        }
        printf("\n");
    }

    return 0;
}